src/DependencyInjection/EzPlatformHttpCacheExtension.php

```diff
@@ -64,4 +68,28 @@ public function getExtraConfigParsers()
     {
         return $this->extraConfigParsers;
     }
+
+    /**
+     * Overrides default header name based on setting tag_mode in FosHttpCacheBundle configuration.
+     */
+    private function setDefaultResponseHeader(ContainerBuilder $container)
+    {
+        $purgeType = $container->getParameter('ezpublish.http_cache.purge_type');
+
+        $responseHeader = 'http' === $purgeType ? 'xkey' : TagHeaderFormatter::DEFAULT_HEADER_NAME;
```

Comment:
Unsure about this, shouldn't it be something like the following? 
```
$responseHeader = in_array($purgeType, ['http', 'varnish', 'fastly']) ? 'xkey' : TagHeaderFormatter::DEFAULT_HEADER_NAME;
```
To cover all `http`-like possibilities? (ref: https://github.com/ezsystems/ezplatform/blob/master/config/packages/ezplatform.yaml#L25) Or did you drop these options and I'm not aware of it perhaps? 

The same question applies to https://github.com/ezsystems/ezplatform-http-cache/pull/92/files#diff-321f92f0967bd338c1a1e6404a0bb29fR91.

Reply:
Yeah, I was not sure about those, `fastly` should probably taken care of in it own package.

Not sure about having dual configuration for `varnish/http`, from looking around the code and docs aside from comment above parameter we are using `http` as an valid purge type.  Should we definitely drop one or support both?

Reply:
As for me, it's a good idea to take this opportunity and clean up these options. I'm good with keeping only `http`, but maybe there is some reason for keeping both. 
@andrerom do you have any input, perhaps?

Reply:
is `http` a generic enough option?
maybe we should rather end up with locale, varnish and fastly?
@vidarl Didn't you introduce the Varnish alias when you added Fastly in order to more clearly separate them?

Reply:
Yes, I also think we should go with `local` (or maybe even rename it to `symfony` to be consistent with fos?), `fastly` and `varnish` as `xkey` is very specific to it, right?

>is http a generic enough option? Will it work on any kind of proxy? :)

I think yes, as long as it accepts PURGE requests and tag invalidation and expects `xkey`. But don't quote me on that ;)

Reply:
> (or maybe even rename it to symfony to be consistent with fos?),

if they call it that, I'm all for it.

> I think yes, as long as it accepts PURGE requests and tag invalidation and expects xkey. But don't quote me on that ;)

Afaik there is currently none, but I might be wrong ;)
LiteSpeed: https://github.com/FriendsOfSymfony/FOSHttpCache/pull/444/files#diff-83872f222995ee0fbb9a54355f54c5c4R70
Fastly: https://github.com/FriendsOfSymfony/FOSHttpCache/pull/451/files#diff-3584dcd2a732e6f34941b9194c2ff416R50

Reply:
Then I am also for `symfony`, `varnish` and `fastly` but I would prefer to change those in separate PRs.